Tools/autotest/common.py

```diff
@@ -3551,6 +3627,18 @@ def validator(value2, target2=None):
 
         self.wait_and_maintain(value_name="Distance", target=distance_min, current_value_getter=lambda: get_distance(), validator=lambda value2, target2: validator(value2, target2), accuracy=(distance_max - distance_min), timeout=timeout, **kwargs)
 
+    def wait_distance_to_home(self, distance_min, distance_max, timeout=10, use_cached_home=True, **kwargs):
+        """Wait for flight of home."""
```

Comment:
Comment needs help.
```suggestion
        """Wait for distance to home to be within specified bounds."""
```

Reply:
damned, I miss this one on the previous PR ...